packages/fluentui/react/src/components/Input/Input.tsx

```diff
@@ -202,7 +202,7 @@ class Input extends AutoControlledComponent<WithAsProp<InputProps>, InputState>
     const { value } = this.state;
 
     if (clearable && (value as string).length !== 0) {
```

Comment:
Now it should be something like:

```
if (clearable && (value as string).length !== 0) {
  return <Box styles />
}

return Icon.create(icon)
```

Reply:
We cannot have children in the Icon component

Reply:
I see what you mean, but it's really weird